test/KBaseReport_server_test.pl

```diff
@@ -13,7 +13,7 @@
 my $config = new Config::Simple($config_file)->get_block('KBaseReport');
 my $ws_url = $config->{"workspace-url"};
 my $ws_name = undef;
-my $ws_client = new Bio::KBase::workspace::Client($ws_url,token => $token);
+my $ws_client = Workspace::WorkspaceClient->new($ws_url,token => $token);
 my $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1);
```

Comment:
hmm- it worked enough to save to CI workspace, but I'll switch to do use the better pattern.